dist/index.js

```diff
@@ -41248,48 +41248,29 @@ const run = async () => {
   core.startGroup('Extracting action context');
 
   // Setup Runner Context
-  const envRunnerFilters = [
-    // Don't strip RUNNER_ and GITHUB_ env vars so we can avoid naming conflicts
-    { prefix: 'RUNNER_', strip: false },
-  ];
+  // Don't strip RUNNER_ and GITHUB_ env vars so we can avoid naming conflicts
   const runnerKey = 'RUNNER_TRACKING_ID';
-
   const githubRunnerCtx = process.env[runnerKey]
     ? {
-        GithubRunner: {
-          key: process.env[runnerKey],
-          ...createContext(envRunnerFilters, runnerKey),
-        },
+        GithubRunner: createContext(process.env[runnerKey], { prefix: 'RUNNER_', strip: false }, runnerKey),
       }
     : {};
 
   // Setup Github Context
-  const envGithubFilters = [{ prefix: 'GITHUB_', strip: false }];
   const githubKey = 'GITHUB_REPOSITORY';
   const githubCtx = process.env[githubKey]
     ? {
-        Github: {
-          key: process.env[githubKey] ? process.env[githubKey].split('/').pop().trim() : process.env['GITHUB_RUN_ID'],
```

Comment:
This branch is part of a ternary that only executes if `process.env[githubKey]` evaluates to true, so we can simplify some of this logic.